sonarqube/resource_sonarqube_portfolio_test.go

```diff
@@ -247,34 +248,6 @@ func TestAccSonarqubePortfolioSelectionModeNone(t *testing.T) {
 	})
 }
 
-func TestAccSonarqubePortfolioSelectionModeManual(t *testing.T) {
-	rnd := generateRandomResourceName()
-	name := "sonarqube_portfolio." + rnd
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:  func() { testAccPreCheck(t); testAccPreCheckPortfolioSupport(t) },
-		Providers: testAccProviders,
-		Steps: []resource.TestStep{
-			{
-				Config: testAccSonarqubePortfolioConfigSelectionMode(rnd, "testAccSonarqubePortfolioKey", "testAccSonarqubePortfolioName", "testAccSonarqubePortfolioDescription", "public", "MANUAL"),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(name, "key", "testAccSonarqubePortfolioKey"),
-					resource.TestCheckResourceAttr(name, "selection_mode", "MANUAL"),
-				),
-			},
-			{
-				ResourceName:      name,
-				ImportState:       true,
-				ImportStateVerify: true,
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(name, "key", "testAccSonarqubePortfolioKey"),
-					resource.TestCheckResourceAttr(name, "selection_mode", "MANUAL"),
-				),
-			},
-		},
-	})
-}
-
```

Comment:
Removing this testcase because it doesn't contain any `selected_projects`, which with the new changes causes a crash. This testcase is covered by the new testcases, in particular `TestAccSonarqubePortfolioManualImport`